### PR TITLE
Migrate `ParVec`

### PR DESCRIPTION
**Pull request by abonander**
Tests and benchmarks run fine. No dependencies besides `std` and a few crates for convenience in the tests, which are only pulled in via the `dev-dependencies` class. Benchmarks look just as good as when I submitted the original PR.

### CODE REVIEW

**Comment by FlashCat** [comment]
Thanks for the pull request, and welcome! The contain-rs team is excited to review your changes, and you should hear from @Gankro (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. The way Github handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

File: Cargo.toml [hunk 1]
```diff
@@ -2,4 +2,11 @@
 
 name = "par-vec"
 version = "0.0.1"
-authors = ["Alexis <a.beingessner@gmail.com>"]
+authors = [
+	"Alexis <a.beingessner@gmail.com>", 
+	"Austin <austin.bonander@gmail.com>"
```
Comment (apasel422) [inline Cargo.toml]: Full names?

Comment (Gankra) [inline Cargo.toml]: Also I wouldn't consider myself an author; this was just generated when I stubbed the file

File: Cargo.toml [hunk 2]
```diff
@@ -2,4 +2,11 @@
...
+]
+
+[dev-dependencies]
+rand = "*"
+threadpool = "*"
```
Comment (Gankra) [inline Cargo.toml:19]: needs at very least the license field; check out the other repos for what we're including in here.

**Comment by abonander** [comment]
@Gankro I have made the requested adjustments. r?

**Comment by Gankra** [comment]
@FlashCat r+

Thanks!

**Comment by FlashCat** [comment]
:pushpin: Commit b6b0728 has been approved by `Gankro`

**Comment by FlashCat** [comment]
:hourglass: Testing commit b6b0728 with merge 43a64b8...

**Comment by FlashCat** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/contain-rs/par-vec/builds/55224389)

**Pull request merged**